The Analysis of BRAF protein in Cancer utilising Homology and Alpha Fold Modelling

The BRAF protein is encoded by the B-Raf proto-oncogene (BRAF) gene, which is found on chromosome 7 (7q34) in the human genome. BRAF has the highest activation capacity among the RAF proteins, although MEK (MEK1 and MEK2) can also be phosphorylated by them. Therefore, in response to extracellular stimuli like cytokines, growth factors, hormones, and environmental stressors, the Ras-Raf-MEK-ERK pathway, also known as the mitogen activated protein kinase/extracellular signal regulated kinase (MAPK/ERK) signalling pathway, is stimulated and thereby regulates cell proliferation, differentiation, and apoptosis. Numerous target genes are known to be stimulated in their expression upon activation of the Raf-MEK-ERK pathway (Kondoh K.,2006). 

As part of its normal job, BRAF controls the MAPK/ERK pathway, which in turn regulates cell division and proliferation. Different domains contribute to its function and control, and its structure is highly adjusted to provide appropriate activation and regulation. These mechanisms are disturbed by BRAF misregulation or mutation, including the V600E mutant, which frequently results in cancer. 

BRAF Protein:

![braf (apo)](https://github.com/user-attachments/assets/34085856-d6b2-4ab4-b3de-e356bd4600d3)

*Figure 1 **apo** form of **RAF** (specifically **BRAF**) using PDB ID **4WO5** in PyMOL,* *"apo" refers to a **protein in its unbound state**, meaning it is **not bound** to any ligand, substrate, cofactor, or drug.* *The kinase domain (chain A) of BRAF includes the key residues involved in the catalysis of the phosphorylation process. Important residues in the kinase domain of BRAF are important in regulation, catalysis, and ATP binding. These residues are important because they are necessary for activity in the active conformation and are visible even in the apo state. Asp594 and Phe595 are responsible for catalytic function (red), Gly-rich loop(blue) for ATP binding. The active site is located on Asp576.*

![braf (agonist)](https://github.com/user-attachments/assets/df1a731c-4e58-45d8-8047-2b1ff1fdb4f3)

*Figure 2 BRAF agonist conformation using PDB ID:4MNE in PyMol,* *an agonist is a molecule (often a drug or endogenous compound) that binds to a receptor and **activates it**, leading to a biological response. Chains A, D, E, and H belong to the Dual specificity mitogen-activated protein kinase 1(MAP2K1). Chains B, C, F, G belong to Serine/threonine-protein kinase B-raf. The structure has 4 unique Ligands namely ACP, 573, CL, MG. The catalytic site (active site) on ASP-576.*

![braf (antagonist)](https://github.com/user-attachments/assets/31aa5018-8099-4da2-8d82-e56c49d4f8b1)

*Figure 3 BRAF Antagonist conformation using PDB ID: 3IDP in Pymol,* *An antagonist is a molecule that binds to a receptor but **does not activate it**. Instead, it **blocks or inhibits** the action of an agonist. There are two chains A and B each with an active site (ASP-576)*







Homology Modelling (SWISS) Structure:

![BRAF (SWISS)](https://github.com/user-attachments/assets/2040f3d1-843b-4fc8-8032-421db99b6afd)

*Figure 4 Homology Modelling Structure generated by SWISS Model, it shows Chain A and Chain B, with two ACP ligands. In Apo conformation.* 

Alpha Fold Modelling Structure:

![BRAF (Alpha)](https://github.com/user-attachments/assets/7e67d801-80e5-44cb-9cad-1e91811624d6)

*Figure 5 Alpha Fold generated BRAF Structure, In apo conformation.* 




Comparison of Structures:

![apo vs SWISS](https://github.com/user-attachments/assets/e3b5b7ea-44f5-4a98-87e8-0f4f8911ddee)

*Figure 6 BRAF apo conformation aligned with SWISS model, RMSD: 24.279*

![agonist vs SWISS](https://github.com/user-attachments/assets/57913735-03bb-4bac-a3d0-2db5c5579842)


*Figure 7 BRAF Agonist conformation aligned with SWISS Model, RMSD: 0.922* 

![antagonist vs SWISS](https://github.com/user-attachments/assets/62f9bbf0-95a9-4732-913c-21ca76abf584)

*Figure 8 BRAF antagonist conformation aligned with SWISS Model, RMSD: 24.399*

![alpha vs apo](https://github.com/user-attachments/assets/e343ba75-f551-4d6c-8a09-3f8c76bf9f8a)

*Figure 9 BRAF apo conformation aligned with Alpha Fold conformation, RMSD:25.108*

![alpha vs agonist](https://github.com/user-attachments/assets/3a30e3ef-1df0-463c-8094-71267c3b4f21)

*Figure 10 BRAF agonist conformation aligned with Alpha fold model, RMSD:0,564*

![alpha vs antagonist](https://github.com/user-attachments/assets/42dda527-09a4-4218-acb2-b79f6ffed773)

*Figure 11 BRAF antagonist conformation aligned with Alpha Fold Model, RMSD: 24,781*

The alpha fold model is the most accurate modelling technique as it produced the smallest RMSD value (0,564) when aligned to the Agonist conformation of BRAF, meaning it predicted the structure that is closes to the experimental structure. The two modelling techniques are both useful but SWISS model gives a clearer and more concise format of the sequence information, which is quite useful if one is not so familiar with the protein structure. The two modelling techniques both generate an Apo conformation of the BRAF protein. Alpha fold does not give much information on the sequence of the structure it generates aside from the actual structure which can be viewed on pymol and used to find interactions in the protein, but less work is better. As scientists we do strive for tool and techniques that are accurate no matter how tedious the process may be, that is why Alpha fold can be deemed the most suitable tool to visualise the BRAF protein.



Reference List:

1. . Kondoh K., Nishida E. Regulation of MAP Kinases by MAP Kinase Phosphatases. *Biochim. Biophys. Acta.* 2007;1773:1227–1237. doi: 10.1016/j.bbamcr.2006.12.002.

